src/utils/subscriptions.js

```diff
@@ -54,7 +91,19 @@ function createSubscriptionStatus(subscription, type) {
  * @returns axios post promise
  */
 function sendNotification(resources, subscription) {
-  const subscriptionStatus = createSubscriptionStatus(subscription);
+  // Update number events since start
+  subscription.numEventsSinceStart += resources.length;
```

Comment:
Should this be  `+= 1` because we only sent one notification?

Reply:
I don't think so. R5 subscriptions (and the backport) allow multiple triggering resources to be sent in one notification. As I've been working with the backport stuff I've made sure to keep that capability. I don't think we would ever get multiple resources but I figured we might as well stay consistent with the spec instead of just assuming. I believe these triggering resources are what the backport calls `events`, hence the `eventsInNotification` parameter to specify how many triggering resources are included in the notification bundle.